### ISSUE TITLE
Import: default ignore pattern to broad

### ISSUE THREAD

**Issue by wordofglass** [beetbox/beets#1972]
I attempted to import a folder containing the albums "…Hits" and "...But Seriously" (notice that Hits has U+2026, but the other album only three dots). Because of http://beets.readthedocs.io/en/v1.3.17/reference/config.html#ignore, beets will _silently_ ignore the "But Seriously" folder.
(This probably not only applies to folders, but also to filenames: If someone previously tagged their files "$album - $track - $title", which I don't, but I imagine there might be people who do).

I consider this to be a significant problem; in a larger import, I would likely have missed this, and probably deleted the original folder at some point. So, it would be nice to sophisticate the default ignore patterns, and make beets more verbose about ignoring folders (at least when they contain files that could be audio).

There are more problematic cases, though. See

http://musicbrainz.org/release-group/eb13b70f-02f8-3c89-a4a9-0418252b0fdf

So just changing the pattern to `.[!.]*` is not really _the_ solution, either.

If someone had a good idea on how to improve the behaviour here, I'd really appreciate it.

**Comment by jackwilsdon** [comment]
Hmm. The issue here is that files beginning with dot are usually hidden files on Unix-like operating systems, so I'm not entirely sure if we'd want to import those.

**Comment by sampsyo** [comment]
Yeah, this is a tricky issue. In the prehistory of beets, before we had the ignore feature at all, we also ran into lots of problems where OS-generated hidden files would get imported as if they were real music. This generated a lot of confusion.

So we're kind of stuck between two unsavory alternatives: allowing leading-dot files has problems, but so does ignoring them by default. And sadly, experience shows that making the decision OS-dependent (e.g., only ignore the leading dot on Unixes, not on Windows) has problems too because people so often keep their music on network filesystems hosted on a different OS.

**Comment by jrobeson** [comment]
couldn't we change it to ignore 2 dots or less? not that it's a perfect fix, but it would cover most cases problably.

**Comment by jackwilsdon** [comment]
That would be a partial fix but there's still albums named like this: https://musicbrainz.org/release-group/eb13b70f-02f8-3c89-a4a9-0418252b0fdf

**Comment by jrobeson** [comment]
well, I don't know what we can do about those, but allowing `...` is a thing we can do right now.
